### ISSUE TITLE
[BG] Bandera de la alianza/Map

### ISSUE THREAD

**Issue by F4NT4SM4** [RoDNet/Bugtracker-Infinium-4.3.4#4646]
Nombre: Bandera de la alianza en cumbres gemelas y garganta grito de guerra
Descripcion: En el map deberia de salir el icono de la bandera si un enemigo la lleva
Problema: No se ve la bandera a veces salen uno pixeles borrosos o el icono del jugador enemigo este problema solo pasa con la bandera de la alianza la bandera de la horda se ve perfectamente

**Comment by j0c54n** [comment]
Cierto, esa es una gran desventaja para nosotros por que ellos pueden ver por donde llevamos su bandera mientras que nosotros no podemos ver por donde esta nuestra bandera.
